cluster/src/main/java/org/apache/iotdb/cluster/server/member/RaftMember.java

```diff
@@ -1334,7 +1338,7 @@ private TSStatus handleLogExecutionException(
           .getStatus(Arrays.asList(((BatchInsertionException) cause).getFailingStatus()));
     }
     TSStatus tsStatus = StatusUtils.getStatus(StatusUtils.EXECUTE_STATEMENT_ERROR,
-        cause.getClass().getName() + ":" + cause.getMessage());
+        cause.getMessage());
     if (cause instanceof IoTDBException) {
```

Comment:
This is not relevant to this PR, but it occurs to me that RunTimeExceptions sometimes does not provide useful messages, and it would difficult to locate the problem.
So please add this:
```
if (cause instanceof RunTimeException) {
  logger.error("RuntimeException during executing {}", log , cause);
}
```

Reply:
That's a good idea.